<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NetworkDlg</class>
 <widget class="QDialog" name="NetworkDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>322</width>
    <height>356</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>322</width>
    <height>356</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>322</width>
    <height>356</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Networks</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>320</y>
     <width>301</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="connectBtn">
      <property name="text">
       <string>Co&amp;nnect</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="cancelBtn">
      <property name="text">
       <string>&amp;Cancel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="okBtn">
      <property name="text">
       <string>&amp;Ok</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>301</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="1">
     <widget class="QLineEdit" name="gblNickLE"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Second Nic&amp;kname:</string>
      </property>
      <property name="buddy">
       <cstring>gblNick2LE</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="gblNick2LE"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>&amp;Username:</string>
      </property>
      <property name="buddy">
       <cstring>gblUsernameLE</cstring>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="gblUsernameLE"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Rea&amp;l Name:</string>
      </property>
      <property name="buddy">
       <cstring>gblRealNameLE</cstring>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="gblRealNameLE"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>&amp;Nickname:</string>
      </property>
      <property name="buddy">
       <cstring>gblNickLE</cstring>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>301</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QListWidget" name="networkList"/>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="addBtn">
        <property name="text">
         <string>&amp;Add</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="editBtn">
        <property name="text">
         <string>&amp;Edit</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="removeBtn">
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Networks:</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>gblNickLE</tabstop>
  <tabstop>gblNick2LE</tabstop>
  <tabstop>gblUsernameLE</tabstop>
  <tabstop>gblRealNameLE</tabstop>
  <tabstop>addBtn</tabstop>
  <tabstop>editBtn</tabstop>
  <tabstop>cancelBtn</tabstop>
  <tabstop>okBtn</tabstop>
  <tabstop>networkList</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>okBtn</sender>
   <signal>clicked()</signal>
   <receiver>NetworkDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>306</x>
     <y>335</y>
    </hint>
    <hint type="destinationlabel">
     <x>381</x>
     <y>338</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelBtn</sender>
   <signal>clicked()</signal>
   <receiver>NetworkDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>332</y>
    </hint>
    <hint type="destinationlabel">
     <x>188</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
